sphinx/prod-server.rst

```diff
@@ -105,6 +106,13 @@ Static Files
         integreat-cms-cli collectstatic
         exit
 
+    5. Set up cronjobs to regularly delete outdated PDF and XLIFF files::
+
+        crontab -e
+
+       ::
+
+        0 0 * * * /usr/bin/find /var/www/integreat-cms/{pdf/*,xliff/{download,upload}/*} -mtime +7 -delete
```

Comment:
@svenseeberg this should do the trick, right?

I want to delete all dirs & files in the directories `/var/www/integreat-cms/pdf/`, `/var/www/integreat-cms/xliff/download/` and `/var/www/integreat-cms/xliff/upload/`, but not the directories itself.

Reply:
Ok, this command is probably unsuitable because it throws an error when there are no files in one of these directories (even if there are files in the other ones):
```
$ /usr/bin/find /var/www/integreat-cms/{pdf/*,xliff/{download,upload}/*
/usr/bin/find: ‘/var/www/integreat-cms/{pdf/*,xliff/download/*’: No such file or directory
```